### DISCUSSION TITLE
Is there a way to add a control to the center of the title bar using WindowCommands?

### DISCUSSION THREAD

**Discussion by TheGameVlog** [MahApps/MahApps.Metro#4125] [Q&A]
Workaround is using RightWindowCommands and then giving it some margins, but then it's not a perfect solution so was wondering if someone else has done it before or has an easier way to do it. 

Thank you.

**Comment by timunie** [comment]
Hi @TheGameVlog 

I would consider using the `TitleTemplate` instead: 

```xaml
<mah:MetroWindow.TitleTemplate>
    <DataTemplate>
        <mah:WindowCommands HorizontalAlignment="Center" ShowLastSeparator="False">
            <Button>😎</Button>
            <Button>💹</Button>
            <Button>🎉</Button>
        </mah:WindowCommands>
    </DataTemplate>
</mah:MetroWindow.TitleTemplate>
```

![image](https://user-images.githubusercontent.com/47110241/121865682-892b2e00-ccfe-11eb-9557-9b06dbbc0021.png)

Happy coding
Tim
